ovs/lib/albamigration.py

```diff
@@ -239,6 +241,38 @@ def migrate():
             except Exception:
                 AlbaMigrationController._logger.exception('Updating auto cleanup failed')
 
+        ###################################################
+        # Sync all disks and apply the backend role. Backend role was removed with the AD (since 1.10)
+        albanode_backend_role_sync_key = '/ovs/framework/migration|albanode_backend_role_sync'
+        if not Configuration.get(key=albanode_backend_role_sync_key, default=False):
+            try:
+                errors = []
+                for alba_node in AlbaNodeList.get_albanodes():
+                    try:
+                        if not alba_node.storagerouter:
+                            continue
+                        stack = alba_node.client.get_stack()  # type: dict
+                        DiskController.sync_with_reality(storagerouter_guid=alba_node.storagerouter_guid)
```

Comment:
Minor remark: place the sync_with_reality code on top of this loop as you are comparing states